public/index.html

```diff
@@ -238,7 +238,6 @@
             <div class="project-name">Scroll Dark Game</div>
             <a href="Scroll Game Dark Run/index.html" class="demo-link">Demo</a>
         </div>
-        <div class="table">
             <div class="day-number">Day 42</div>
             <div class="project-name">Amazon App</div>
             <a href="Amazon_Clone/index.html" class="demo-link">Demo</a>
```

Comment:
fix this 
![image](https://github.com/dhairyagothi/50_days_50_web_project/assets/142989448/8e98a2bf-ffe4-48ce-af8a-6c8bc9d9b007)

pull main branch first

Reply:
@Yaser-123